contracts/upgradeable_contracts/erc20_to_native/ForeignBridgeErcToNative.sol

```diff
@@ -79,6 +82,12 @@ contract ForeignBridgeErcToNative is BasicForeignBridge, ERC20Bridge, OtherSideB
     ) internal returns (bool) {
         setTotalExecutedPerDay(getCurrentDay(), totalExecutedPerDay(getCurrentDay()).add(_amount));
         uint256 amount = _amount.div(10**decimalShift());
+
+        uint256 currentBalance = tokenBalance(erc20token());
+        if (currentBalance < amount) {
+            _convertChaiToDai(amount.sub(currentBalance).add(minDaiTokenBalance()));
```

Comment:
This seems to be an invalid scenario. 
If Chai token is not configured, then bridge should have everything stored in Dai, so it will have enough funds to cover any xDai => Dai transfer, in other words `currentBalance >= amount` in this case.

Reply:
Understood. Please add a comment here to reduce number of comments from the security auditors.